src/env/from_envp.c

```diff
@@ -44,7 +43,6 @@ int						env_from_envp(t_env *env, char **envp)
 			env_list_delete(env);
 			return (-1);
 		}
-		assert(ft_strchr(envp[i], '=') != NULL);
```

Comment:
While the assert is gone, the code still assumes that a `=` is present. So it will now just crash when finding an env without a `=`.

Please change the code so it drops invalid environment variables and maybe even prints a warning if you feel like it. Some tests would also be nice.